package.json

```diff
@@ -162,7 +162,9 @@
     "prismjs": "1.25.0",
     "ansi-regex": "5.0.1",
     "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "react-dom": "^16.14.0",
+    "marked": "^4.0.10",
```

Comment:
> @brbrown25 Thanks for the contribution, but in the current state we cannot merge this PR. We need to solve the issue with `marked` version because those change will break the release process. Please let us know if you will be handling the requested changes or we should take over. Alse please note that we have entire epic dedicated to dependencies upgrade for this repository and we will progress with this soon.

I will run with the requested changes.